content/_data/changelogs/weekly.yml

```diff
@@ -27069,8 +27069,8 @@
           - basil
         pr_title: "[JENKINS-75683] re-introduce actions for Jenkins with custom action.jelly"
         message: |-
-          <code>RootAction</code>s that used custom rendering (<code>action.jelly</code>) are once again visible in the header and behave appropriately.
-          <code>RootAction</code>s that use jelly to show/hide are correctly hidden in the header
+          <code>RootActions</code> that used custom rendering (<code>action.jelly</code>) are once again visible in the header and behave appropriately.
```

Comment:
it would be intentional as the class is `RootAction`

Reply:
Ah, the third line includes the s, and when rendered on the site it looks inconsistent, so I was going off that.
<img width="1096" alt="Screenshot 2025-06-24 at 11 01 38 AM" src="https://github.com/user-attachments/assets/829ff9b1-6866-4d1a-9f57-f9d0f955d30c" />

Reply:
```suggestion
          Any <code>RootAction</code> that used custom rendering (<code>action.jelly</code>) is once again visible in the header and behaves appropriately.
```
Would this still make sense while avoiding the formatting issue?